users/static/user_site/home.js

```diff
@@ -0,0 +1,23 @@
+var toggle = document.querySelector('.toggle');
+var calender = document.querySelector('.calender');
+
+toggle.addEventListener('click', (e)=> {
+toggleClass(calender, 'cal-view');
+});
+
+let toggleClass = (el, className) => {
+if (el.classList) {
+	el.classList.toggle(className);
+} else {
+	var classes = el.className.split(' ');
+	var existingIndex = classes.indexOf(className);
```

Comment:
Maybe you can use `const` or `let` instead of `var`

Reply:
done. thanks 😄